### ISSUE TITLE
"close" button for notification/alert messages – include in tab order

### ISSUE THREAD

**Issue by ekate** [inveniosoftware/invenio-app-rdm#1192]
From NYU A11y Review:

Button to dismiss notifications cannot be reached by Assisted Technology.

The button to dismiss the notification/alert message cannot be reached by assistive technology users (screen reader/keyboard-only) as it is not included in the tab order.
